app/modules/container/container.go

```diff
@@ -58,31 +58,31 @@ func ExecDockerPush(ctx *context.LedoContext, command cli.Args) {
 	} else {
 		version = command.First()
 	}
-	image := ShowDockerImageFQN(ctx)
+	image := ShowImageFQN(ctx)
 	args = append(args, "push")
 	args = append(args, image+":"+version)
-	err := ctx.ExecCmd("docker", args[0:])
+	err := ctx.ExecCmd(ctx.GetRuntimeCommand(), args[0:])
 	if err != nil {
 		return
 	}
 }
 
-func ExecDockerRetag(ctx *context.LedoContext, command cli.Args) {
+func ExecRetag(ctx *context.LedoContext, command cli.Args) {
```

Comment:
Nie lepiej `ExecTag`? `retag` bardziej sugeruje, że "zmieniamy" tag a tak naprawdę powstaje nowy :}

Reply:
To jest retagowanie, więc ja bym tak zostawił. Druga rzecz to zachowanie kompatybilności wstecznej.